lib/tools.js

```diff
@@ -396,7 +399,12 @@ function final (logger, handler) {
 
 function stringify (obj) {
   try {
-    return JSON.stringify(obj)
+    return JSON.stringify(obj, (key, value) => {
+      if (typeof value === 'bigint') {
+        return value.toString() + 'n'
+      }
+      return value
+    })
```

Comment:
This is going to slow us down significanfly. I'm -1 to us landing this feature.

Reply:
The deep object benchmarks confirm this.

Reply:
>This is going to slow us down significanfly.

have you checked the case then JSON.strigify throws an error? I bet it goes down a lot.

>I'm -1 to us landing this feature.

and -{a lot of users} because luck of bigint support.

––––

I guess it's possible to cofigure pino to skip bigint check. does it sound fine for you guys?

Reply:
Thanks for the pull request. We have had a discussion and have determined that `BigInt` needs better support in general. This implementation negatively impacts our benchmarks even when a `BigInt` is not present.